### ISSUE TITLE
[Feature request] Add option to boot game with save state in Custom Configuration per game for setups without a mouse or keyboard

### ISSUE THREAD

**Issue by TheHyeOne** [RPCS3/rpcs3#13004]
### Quick summary

Add the ability to add save states on boot as an option in the custom configuration per game.

### Details

I would like the ability to add save states as an option in the custom configuration per game.  Users who have light gun setups and arcade cabinets will not be able to use the ability to use save states at all using a joystick/buttons.  Currently you can right click a game and boot with save state, but cannot set the game to do the exact same function as an option in the custom configuration per game.  Adding it would bring a lot of great functionality to more users.

Would be a UI Change and fully functional via right click in the menu already.

**Comment by Megamouse** [comment]
This would make more sense as a cli command

**Comment by Megamouse** [comment]
I think this is obsolete, since you can:
- Navigate the UI using the controller
- Boot savestates via command line options

**Comment by TheHyeOne** [comment]
This is more for those with arcade setups that boot directly into a game
through a front end like launchbox, would be much easier to have a checkbox
for this and you can set it up while you do your per game config, imo

**Comment by digant73** [comment]
@elad335 is this FR still valid or maybe it has been covered by other better solutions and so this fr can be considered implemented?

**Comment by elad335** [comment]
This is more of a UX thing so I'll let @Megamouse decide.
There is also #17688

**Comment by digant73** [comment]
with #17688 merged some months ago, this FR could probably be considered implemented and so closed

**Comment by Megamouse** [comment]
Yeah the feature request is very niche and there's already other ways to do it
